editor/header/publish-with-dropdown/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress Dependencies
+ */
+import { Dropdown, Button, Dashicon } from '@wordpress/components';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+import PublishDropdown from '../publish-dropdown';
+import PublishButtonLabel from '../publish-button/label';
+import {
+	isSavingPost,
+	isEditedPostSaveable,
+	isEditedPostPublishable,
+} from '../../selectors';
+
+function PublishWithDropdown( { isSaving, isPublishable, isSaveable } ) {
+	const isButtonEnabled = ! isSaving && isPublishable && isSaveable;
+
+	return (
+		<Dropdown
+			position="bottom left"
+			className="editor-publish-with-dropdown"
```

Comment:
Newbie question, file name is `editor/header/publish-with-dropdown/index.js`. A class name is combined from `editor` keyword and `folder name`. Is that a pattern we use and the reason that `header` subfolder is ignored?

Reply:
Yes, that's how we do it right now :)